app/views/content_items/hmrc_manual.html.erb

```diff
@@ -13,7 +13,7 @@
   <% @content_item.section_groups.each do |group| %>
     <% next unless group["child_sections"].present? %>
     <div class="subsection-collection">
-      <%= render "content_items/manuals/hmrc_sections", group: group %>
+      <%= render "content_items/manuals/hmrc_sections", group: group, heading: 3 %>
```

Comment:
`heading` could be updated to `heading_level` to help make it more descriptive of what it is used for, example of this approach in `frontend` - https://github.com/alphagov/frontend/blob/e897dc6675d8c02118be82ff08d20b92d84d7437/app/views/landing_page/blocks/_heading.html.erb#L13